CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 *   New Features
     *   Add an option in your account settings to modify your avatar with Gravatar
         ([#2263](https://github.com/Automattic/pocket-casts-android/pull/2263))
+    *   Add the ability to rate podcasts
```

Comment:
I am updating the changelog here since we agreed to release this feature in `7.70`. p1721732895524199/1721647137.664459-slack-C077XU4GF9D